src/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import { Person } from './types/Person';
+import { getPeople } from './api';
+import { Loader } from './components/Loader';
+import { NoPeople } from './NoPeople';
+import { LoadingError } from './components/LoadError';
+import { TablePeople } from './components/TablePeople';
+
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [newError, setNewError] = useState(false);
+  const [load, setLoad] = useState(true);
```

Comment:
Consider adding a generic type here for code consistency

Reply:
i don't think that it'll be more useful than this